tools/generate_proto_go.sh

```diff
@@ -4,8 +4,8 @@ PROJECT_ROOT="$(pwd)/$(dirname "$0")/../"
 
 QDRANT_PROTO_DIR='proto'
 
-go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.34.2
-go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.5.1
+go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
```

Comment:
this is trick, are we sure we don't want to pin explicit versions for reproducibility?

Reply:
Yeah. I had the doubt too.
I followed the instructions at https://grpc.io/docs/languages/go/quickstart/#prerequisites

Reply:
I think it's assuming backward compat, latest is recommended.

Reply:
Regardless, let me pin it to a version.

Reply:
[`9659554` (#54)](https://github.com/qdrant/go-client/pull/54/commits/9659554a250be3decdd11386779ec00bb18e6dd4)

Reply:
👍